packages/next/telemetry/trace/report/to-jaeger.ts

```diff
@@ -2,6 +2,9 @@ import retry from 'next/dist/compiled/async-retry'
 import { randomBytes } from 'crypto'
 import fetch from 'node-fetch'
 import * as Log from '../../../build/output/log'
+import util from 'util'
```

Comment:
Remove unused from current line to fix linting error.

> /home/runner/work/next.js/next.js/packages/next/telemetry/trace/report/to-jaeger.ts
5:8  warning  'util' is defined but never used  @typescript-eslint/no-unused-vars

```suggestion
```